pyproject.toml

```diff
@@ -0,0 +1,199 @@
+# https://packaging.python.org/guides/single-sourcing-package-version/
+version = "0.4.1.dev1"  # Required
+
+# This is a one-line description or tagline of what your project does. This
+# corresponds to the "Summary" metadata field:
+# https://packaging.python.org/specifications/core-metadata/#summary
+description = "A metrics toolkit to evaluate neural network explanations."  # Optional
+
+# This is an optional longer description of your project that represents
+# the body of text which users will see when they visit PyPI.
+#
+# Often, this is the same as your README, so you can just read it in from
+# that file directly (as we have already done above)
+#
+# This field corresponds to the "Description" metadata field:
+# https://packaging.python.org/specifications/core-metadata/#description-optional
+readme = "README.md" # Optional
+
+# Specify which Python versions you support. In contrast to the
+# 'Programming Language' classifiers above, 'pip install' will check this
+# and refuse to install the project if the version does not match. See
+# https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires
+requires-python = ">=3.7"
+
+# This is either text indicating the license for the distribution, or a file
+# that contains the license
+# https://packaging.python.org/en/latest/specifications/core-metadata/#license
+license = { file = "LICENSE" }
+
+# This field adds keywords for your project which will appear on the
+# project page. What does your project relate to?
+#
+# Note that this is a list of additional keywords, separated
+# by commas, to be used to assist searching for the distribution in a
+# larger catalog.
+keywords = ["explainable ai", "xai", "machine learning", "deep learning"]  # Optional
+
+# This should be your name or the name of the organization who originally
+# authored the project, and a valid email address corresponding to the name
+# listed.
+authors = [
+    { name = "Anna Hedstrom", email = "hedstroem.anna@gmail.com" } # Optional
+]
+
+# This should be your name or the names of the organization who currently
+# maintains the project, and a valid email address corresponding to the name
+# listed.
+maintainers = [
+    { name = "Anna Hedstrom", email = "hedstroem.anna@gmail.com" } # Optional
+]
+
+# Classifiers help users find your project by categorizing it.
+#
+# For a list of valid classifiers, see https://pypi.org/classifiers/
+classifiers = [# Optional
+    # How mature is this project? Common values are
+    #   3 - Alpha
+    #   4 - Beta
+    #   5 - Production/Stable
+    "Development Status :: 5 - Production/Stable",
+    # Indicate who your project is intended for
+    "Intended Audience :: Education",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    # Pick your license as you wish
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    # "Topic :: Scientific :: Explainable Artificial Intelligence",
```

Comment:
Why commented out?

Reply:
It caused an error, while uploading to test-pypi:
```
HTTPError: 400 Bad Request from https://test.pypi.org/legacy/                                                                                                            
Invalid value for classifiers. Error: Classifier 'Topic :: Scientific :: Explainable Artificial Intelligence' is not a valid classifier. 
```

Reply:
> Why commented out?

It was causing an error while uploading to test pypi:
```
HTTPError: 400 Bad Request from https://test.pypi.org/legacy/ Invalid value for classifiers. 
Error: Classifier 'Topic :: Scientific :: Explainable Artificial Intelligence' is not a valid classifier.